### DISCUSSION TITLE
:mega: Want to join the stm32duino team?

### DISCUSSION THREAD

**Discussion by fpistm** [stm32duino/Arduino_Core_STM32#2555] [Q&A]
Hi STM32duino community, 

We are looking for new contributor/maintainer to help maintain the stm32duino community.
If you are interested to join the stm32duino team to contribute to one or more repositories.
Feel free to comment this discussion.

## :sparkles: How you can contribute

* __Triage__ open issues: try to reproduce issues reported by other users and confirm whether you can experience them as well, or ask users for more details if needed. Spot duplicates. Improve descriptions. Help users who ask for support.
* __Submit__ fixes and implementations: pick an open issue or feature request that you think you can implement yourself, and submit a pull request.
* __Review__ and comment on the changes proposed in open pull requests suggesting good ways to implement fixes and features.
* __Test__ open pull requests: try to run the proposed modifications and report your success or failure. Testing on real hardware takes time and any help in this will speed up our responsiveness in merging contributions.
* __Help others__ by answering to GitHub [discussions] or topics in the dedicated [stm32duino forum]
* __Write documentation__ and improve the existing content. Including the [WiKi]

[discussions]: https://github.com/orgs/stm32duino/discussions
[stm32duino forum]: https://www.stm32duino.com
[WiKi]: https://github.com/stm32duino/Arduino_Core_STM32/wiki

**Comment by rmzmrnn** [comment]
Hi 👋
I came across this post and wanted to ask if you're still looking for contributors or maintainers for this project. I'm interested in helping out and would love to know how I can get started if there's still an opportunity. Thanks!

**Reply by fpistm** [reply]
Hi @rmzmrnn 
It is always open.
There is a lot of topics. 
What would interest you? Adding variants? Issue resolution? review?.... Implementing new feature?....

**Reply by rmzmrnn** [reply]
That’s great to hear! I’ll explore the discussions and repositories, and if anything catches my interest, I’ll do my best to contribute.

I currently only have the NUCLEO-F401RE board, so my contributions may focus on that platform. Still, I’ll do my best to help out with other issues and discussions as well.
